docs/src/examples/components/Dropdown/Types/DropdownExampleMultipleSelection.shorthand.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Dropdown } from '@stardust-ui/react'
+
+const inputItems = [
+  'Bruce Wayne',
+  'Natasha Romanoff',
+  'Steven Strange',
+  'Alfred Pennyworth',
+  `Scarlett O'Hara`,
+  'Imperator Furiosa',
+  'Bruce Banner',
+  'Peter Parker',
+  'Selina Kyle',
+]
+
+const DropdownExampleMultipleSelection = () => (
+  <Dropdown
+    multiple
+    items={inputItems}
+    placeholder="Select your heroes"
+    noResultsMessage="We couldn't find any matches."
```

Comment:
Do we need this? Will this be shown if the dropdown doesn't have items? I thought the usage is when the user search for something and there is not result...

Reply:
used for both scenarios:
- no items in the initial list
- no items matching search query